ldi-core/ldes-client/pom.xml

```diff
@@ -18,6 +18,7 @@
         <module>tree-node-fetcher</module>
         <module>tree-node-supplier</module>
         <module>tree-node-relations-fetcher</module>
+        <module>event-stream-properties-fetcher</module>
```

Comment:
like i mentioned, this being a separate module is not my most ideal solution, but for this PR, i agree on the approach